controllers/nodefeaturediscovery_controls.go

```diff
@@ -35,6 +37,8 @@ type ResourceStatus int
 const (
 	Ready ResourceStatus = iota
 	NotReady
+
+	servicePort int = 12000
```

Comment:
Suggest slight renaming:

```suggestion
	defaultServicePort int = 12000
```